### PR TITLE
Doaa Alalawi

### PR DESCRIPTION
**Pull request by doaaAlalawi**

### CODE REVIEW

**Comment by doaaAlalawi** [comment]
on a scale from 1 to 5, how comfortable were you with this assignment?
5
How long did it take?
about an hour
What was a challenge you had with this assignment? 3 and 4 
Is there anything that you'd like some further information on?
No, thanks.
Do you have any suggestions to improve this assignment?
No.

File: HW.rb [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+  # Bonus
+  puts "Bonus"
+  
+  friend = " "
+friends = []
+while friend != "n" do
+    p "add a student" 
+    friend = gets.chomp
+    friends.push(friend)
+    p "Do you want to continue ? (y/n)"
```
Comment (yasser9947) [inline HW.rb:210]: what happend if i type some letter not Y or N ?

**Review by yasser9947 — COMMENTED** [review]
Very great work Doaa
What is creativity? Smart solution
I think you have the skill of learning speed so you gain knowledge and apply it, so I am not afraid for you, you have an impressive future, and also your development is very noticeable 👍 
Greetings H3ro

File: HW.rb (hunk 1 above)
Comment (doaaAlalawi) [reply HW.rb:210]: mmm I tried to do it like this ... 
while friend != "n" || friend != "No" || friend != "no" ||friend != "NO"
but unfortunately it doesn't work probably with me

**Pull request closed without merging**